docs/FAQ.md

```diff
@@ -89,16 +89,11 @@ following conditions:
 will use a bottled version of the formula, but
 `brew install <formula> --enable-bar` will trigger a source build.
 * The `--build-from-source` option is invoked.
-* The environment variable `HOMEBREW_BUILD_FROM_SOURCE` is set.
```

Comment:
What about adding something that this is not intended for general use? Like `(for developers only)`.